zerver/models.py

```diff
@@ -1467,6 +1467,7 @@ class UserBaseSettings(models.Model):
     fluid_layout_width: bool = models.BooleanField(default=False)
     high_contrast_mode: bool = models.BooleanField(default=False)
     translate_emoticons: bool = models.BooleanField(default=False)
+    display_emoji_reaction_users: bool = models.BooleanField(default=False)
```

Comment:
I believe we want a default of `True` here -- i.e. we would display the names by default.

@alya to confirm, but I'm going to adjust this in my update on this PR.

Reply:
Hm, I don't recall whether we've discussed it, but that sounds right. We believe this to be a good experience, and it makes the feature *much* more discoverable.

Reply:
OK, changed. It's even potentially possible that we'll end up removing the setting, if nobody toggles it off.